### ISSUE TITLE
Cant run pipupgrade. ModuleNotFoundError: No module named 'ctypes.windll'

### ISSUE THREAD

**Issue by dusty-dusty** [achillesrasquinha/pipupgrade#39]
ModuleNotFoundError: No module named 'ctypes.windll'

Traceback (most recent call last):
  File "c:\program files\python37\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\program files\python37\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Program Files\Python37\Scripts\pipupgrade.exe\__main__.py", line 5, in <module>
  File "c:\program files\python37\lib\site-packages\pipupgrade\__init__.py", line 7, in <module>
    from pipupgrade.__main__ import main
  File "c:\program files\python37\lib\site-packages\pipupgrade\__main__.py", line 5, in <module>
    from pipupgrade.commands import command as main
  File "c:\program files\python37\lib\site-packages\pipupgrade\commands\__init__.py", line 9, in <module>
    from pipupgrade.commands.helper import (
  File "c:\program files\python37\lib\site-packages\pipupgrade\commands\helper.py", line 8, in <module>
    from pipupgrade.commands.util       import cli_format
  File "c:\program files\python37\lib\site-packages\pipupgrade\commands\util.py", line 2, in <module>
    from pipupgrade.cli.parser import get_args
  File "c:\program files\python37\lib\site-packages\pipupgrade\cli\__init__.py", line 3, in <module>
    from pipupgrade.cli.parser import get_args
  File "c:\program files\python37\lib\site-packages\pipupgrade\cli\parser.py", line 25, in <module>
    _cli.format(__name__,        _cli.RED),
  File "c:\program files\python37\lib\site-packages\pipupgrade\cli\util.py", line 52, in format
    kernel32 = import_handler("ctypes.windll.kernel32")
  File "c:\program files\python37\lib\site-packages\pipupgrade\util\imports.py", line 18, in import_handler
    handler = _HANDLER_REGISTRY[name]
  File "c:\program files\python37\lib\site-packages\pipupgrade\util\imports.py", line 8, in __missing__
    module = __import__(module_name, {}, {}, [handler_name])
ModuleNotFoundError: No module named 'ctypes.windll'

**Comment by dusty-dusty** [comment]
pip 19.2.3 from c:\program files\python37\lib\site-packages\pip (python 3.7

**Comment by dusty-dusty** [comment]
Package     Version
----------- -------
keyboard    0.13.4
mouse       0.7.0
mss         4.0.3
numpy       1.17.2
Pillow      6.1.0
pip         19.2.3
pipupgrade  1.5.2
setuptools  39.0.1
tokenize-rt 3.2.0

**Comment by dusty-dusty** [comment]
python 3.7.4

**Comment by bromix** [comment]
same here

**Comment by HimbeersaftLP** [comment]
Can confirm

**Comment by logonoff** [comment]
this fixes the issue: comment out or delete
```py
51    if _CAN_ANSI_FORMAT_WINDOWS:
52        kernel32 = import_handler("ctypes.windll.kernel32")
53        kernel32.SetConsoleMode(kernel32.GetStdHandle(-11), 7)
```
in `python37\lib\site-packages\pipupgrade\cli\util.py`

**Comment by karserasl** [comment]
My understanding is that you can't import ctypes.windll.
You import ctypes and then just use the ctypes.windll
In util.py, Import ctypes and replace  
`kernel32 = import_handler("ctypes.windll.kernel32")`
with `kernel32 = ctypes.windll.kernel32`

Atleast it works like this. 
(Sorry still a beginner)

**Comment by achillesrasquinha** [comment]
This has been fixed in `1.6.4`
